Mage/src/main/java/mage/abilities/effects/common/continuous/CantLoseLifeControllerEffect.java

```diff
@@ -0,0 +1,42 @@
+
+package mage.abilities.effects.common.continuous;
+
+import mage.abilities.Ability;
+import mage.abilities.effects.ContinuousEffectImpl;
+import mage.constants.Duration;
+import mage.constants.Layer;
+import mage.constants.Outcome;
+import mage.constants.SubLayer;
+import mage.game.Game;
+import mage.players.Player;
+
+/**
+ * @author Sidorovich77
+ */
+public class CantLoseLifeControllerEffect extends ContinuousEffectImpl {
+
+    public CantLoseLifeControllerEffect(Duration duration) {
+        super(duration, Layer.PlayerEffects, SubLayer.NA, Outcome.Benefit);
+        staticText = "Your life total can't change. <i>(You can't lose life. You can't pay any amount of life except 0.)</i>";
```

Comment:
wrong text, also, why is this a common class?

Reply:
Can you please explain why usage of common class is mistake? I cant find a different class for this effect, because it doesn't fit for keyword or mana class.

Reply:
It's not a _mistake_, but if an effect is only used by a single card, usually a custom class is made in that card file. It's fine to make a common class if you anticipate that other cards could use it.

Reply:
In this case I think it's fine as a common class as long as you fix the text